app/models/video.rb

```diff
@@ -1,10 +1,22 @@
 class Video < ActiveRecord::Base
-  belongs_to :category
+  before_validation :update_rating!
 
   validates_presence_of :title, :description
+  belongs_to :category
+  has_many :reviews, -> { order "created_at DESC" }
 
   def self.search_by_title(search_string)
     return [] if search_string.blank?
     Video.where('title LIKE ?', "%#{search_string}%").order('created_at DESC')
   end
+
+  private
+
+  def update_rating!
+    if self.reviews.empty?
+      self.rating = 1.0
```

Comment:
This is a really good way to get Sony Pictures calling you mad (obviously a joke, but it would be the truth if you were Netflix) because all of their unrated videos are rated at 1.0...consider allowing it to be nil and showing N/A on the video show page.